lib/spack/spack/package.py

```diff
@@ -565,6 +563,11 @@ def fetcher(self):
     def fetcher(self, f):
         self._fetcher = f
 
+    def dependencies_of_type(self, *deptypes):
```

Comment:
This is for getting *static* info from the `Package` metadata, while `dependencies` is on `Spec`.